src/Components/Header/Header.js

```diff
@@ -8,10 +8,23 @@ export default function Header(props) {
         <>
             <h1>Pokemon Card Collector</h1>
             <NavBar />
-            {loggedInUser > 0 && (
+            <span>{loggedInUser.username}</span>
+
+            <img
+                className="profilePic"
+                src={loggedInUser.profilePic}
+                alt="profile pic for user"
+            />
+
+            {loggedInUser.userID > 0 && (
                 <button
                     onClick={() => {
-                        setLoggedInUser(0);
+                        setLoggedInUser({
```

Comment:
here you could use that default variable I was mentioning in the comment above instead of typing out the whole user object